packages/@aws-cdk/aws-cognito/test/integ.user-pool-idp.ts

```diff
@@ -0,0 +1,42 @@
+import { App, Stack } from '@aws-cdk/core';
+import { OAuthScope, UserPool, UserPoolIdentityProvider } from '../lib';
+
+/*
+ * Stack verification steps - TBD
+ */
```

Comment:
This integ test will be fixed up after this PR is merged - https://github.com/aws/aws-cdk/pull/8155